orchestra/models/core/models.py

```diff
@@ -311,10 +311,12 @@ class Project(ProjectMixin, models.Model):
     class Status:
         ACTIVE = 0
         ABORTED = 2
+        PAUSED = 3
 
     STATUS_CHOICES = (
         (Status.ACTIVE, 'Active'),
-        (Status.ABORTED, 'Aborted'))
+        (Status.ABORTED, 'Aborted'),
+        (Status.PAUSED, 'Paused'))
```

Comment:
this would be bad with respect to our previous implementation, but we might have done it: there might be code that looks for project status__gte=active or something like that. can you grep the codebase for queries that have a project status in them to make sure we don't rely on the relative order of the status values?

Reply:
Fortunately, we didn't. (grep `Project.Status.ACTIVE`)